locator/types.hh

```diff
@@ -10,7 +10,12 @@
 
 #pragma once
 
+#include <unordered_set>
+
+#include <boost/intrusive/list.hpp>
+
```

Comment:
Unnecessary?

Reply:
hmm, yeah, likely a remnant of a past incarnation of the branch.